src/Components/Main.js

```diff
@@ -26,30 +26,70 @@ const Main = () => {
-      <form onSubmit={(e) => e.preventDefault()} className="w-[600px] min-w-[400px] border border-gray-100 p-16 shadow-lg">
+      <form
+        onSubmit={(e) => e.preventDefault()}
+        className="w-[600px] min-w-[400px] border border-gray-100 p-16 shadow-lg"
+      >
         <h3 className="p-4 text-xl">Video Information:</h3>
         <div className="px-4 py-2">
-          <input type="text" value={project} onChange={(e) => setProject(e.target.value)} className="border border-gray-100 w-full p-2" placeholder="Project name" />
+          <input
+            type="text"
+            value={project}
+            onChange={(e) => setProject(e.target.value)}
+            className="border border-gray-100 w-full p-2"
+            placeholder="Project name"
+          />
         </div>
         <div className="px-4 py-2">
-          <input type="text" value={videoURL} onChange={(e) => setVideoURL(e.target.value)} className="border border-gray-100 w-full p-2" placeholder="Video URL" />
+          <input
+            type="text"
+            value={videoURL}
+            onChange={(e) => setVideoURL(e.target.value)}
+            className="border border-gray-100 w-full p-2"
+            placeholder="Video URL"
+          />
         </div>
         <div className="px-4 py-2 grid grid-cols-12">
-          <input type="checkbox" checked={needsTranscoding} onChange={() => setTranscodingNeeds(!needsTranscoding)} name="transcription" value="transcription" />{" "}
-          <label>Transcription</label>
+          <input
+            id="transcription"
+            type="checkbox"
+            checked={needsTranscoding}
+            onChange={() => setTranscodingNeeds(!needsTranscoding)}
+            name="transcription"
+            value="transcription"
+          />{" "}
+          <label for="transcription">Transcription</label>
         </div>
         <div className="px-4 py-2 grid grid-cols-12">
-          <input type="checkbox" checked={needsSubtitles} onChange={() => setSubtitlesNeeds(!needsSubtitles)} name="subtitles" value="subtitles" /> <label>Subtitles</label>
+          <input
+            id="subtitles"
+            type="checkbox"
+            checked={needsSubtitles}
+            onChange={() => setSubtitlesNeeds(!needsSubtitles)}
+            name="subtitles"
+            value="subtitles"
+          />{" "}
+          <label for="subtitles">Subtitles</label>
         </div>
         <div className="px-4 py-2 grid grid-cols-12">
-          <input type="checkbox" checked={needsThumbnail} onChange={() => setThumbailNeeds(!needsThumbnail)} name="thumbnail" value="thumbnail" /> <label>Thumbnail</label>
+          <input
+            type="checkbox"
+            id="thumbnail"
+            checked={needsThumbnail}
+            onChange={() => setThumbailNeeds(!needsThumbnail)}
+            name="thumbnail"
+            value="thumbnail"
+          />{" "}
+          <label for="thumbnail">Thumbnail</label>
```

Comment:
Can we add className="cursor-pointer" to all the labels for better ux ?

Reply:
Done